packages/simlog/simlog.0.0.4/opam

```diff
@@ -0,0 +1,42 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "The mediocre OCaml logging library"
+description: "The mediocre OCaml logging library"
+maintainer: ["Muqiu Han"]
+authors: ["Muqiu Han"]
+license: "MIT"
+tags: ["log" "logging" "logger"]
+homepage: "https://github.com/muqiuhan/simlog"
+doc: "https://github.com/muqiuhan/simlog"
+bug-reports: "https://github.com/muqiuhan/simlog/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "3.5"}
+  "ocolor"
+  "core"
```

Comment:
```suggestion
  "core"
  "core_kernel" {>= "v0.15.0"}
```
due to
```
#=== ERROR while compiling simlog.0.0.4 =======================================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.11.2 | pinned(https://github.com/X-Files-Institute/simlog/archive/refs/tags/v0.0.4.tar.gz)
# path                 ~/.opam/4.11/.opam-switch/build/simlog.0.0.4
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p simlog -j 127 @install
# exit-code            1
# env-file             ~/.opam/log/simlog-7-86496d.env
# output-file          ~/.opam/log/simlog-7-86496d.out
### output ###
# File "/home/opam/.opam/4.11/lib/spawn/spawn.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# (cd _build/default && /home/opam/.opam/4.11/bin/ocamlc.opt -w -40 -g -bin-annot -I lib/.simlog.objs/byte -I /home/opam/.opam/4.11/lib/base -I /home/opam/.opam/4.11/lib/base/base_internalhash_types -I /home/opam/.opam/4.11/lib/base/caml -I /home/opam/.opam/4.11/lib/base/md5 -I /home/opam/.opam/4.11/lib/base/shadow_stdlib -I /home/opam/.opam/4.11/lib/base_bigstring -I /home/opam/.opam/4.11/lib/base_quickcheck -I /home/opam/.opam/4.11/lib/bin_prot -I /home/opam/.opam/4.11/lib/bin_prot/shape -I /home/opam/.opam/4.11/lib/core -I /home/opam/.opam/4.11/lib/core/error_checking_mutex -I /home/opam/.opam/4.11/lib/core_kernel -I /home/opam/.opam/4.11/lib/core_kernel/base_for_tests -I /home/opam/.opam/4.11/lib/core_kernel/caml_unix -I /home/opam/.opam/4.11/lib/core_kernel/flags -I /home/opam/.opam/4.11/lib/core_kernel/version_util -I /home/opam/.opam/4.11/lib/core_unix -I /home/opam/.opam/4.11/lib/fieldslib -I /home/opam/.opam/4.11/lib/jane-street-headers -I /home/opam/.opam/4.11/lib/ocaml/threads -I /home/opam/.opam/4.11/lib/ocolor -I /home/opam/.opam/4.11/lib/parsexp -I /home/opam/.opam/4.11/lib/ppx_assert/runtime-lib -I /home/opam/.opam/4.11/lib/ppx_bench/runtime-lib -I /home/opam/.opam/4.11/lib/ppx_compare/runtime-lib -I /home/opam/.opam/4.11/lib/ppx_enumerate/runtime-lib -I /home/opam/.opam/4.11/lib/ppx_expect/collector -I /home/opam/.opam/4.11/lib/ppx_expect/common -I /home/opam/.opam/4.11/lib/ppx_expect/config -I /home/opam/.opam/4.11/lib/ppx_expect/config_types -I /home/opam/.opam/4.11/lib/ppx_hash/runtime-lib -I /home/opam/.opam/4.11/lib/ppx_here/runtime-lib -I /home/opam/.opam/4.11/lib/ppx_inline_test/config -I /home/opam/.opam/4.11/lib/ppx_inline_test/runtime-lib -I /home/opam/.opam/4.11/lib/ppx_module_timer/runtime -I /home/opam/.opam/4.11/lib/ppx_sexp_conv/runtime-lib -I /home/opam/.opam/4.11/lib/sexplib -I /home/opam/.opam/4.11/lib/sexplib/unix -I /home/opam/.opam/4.11/lib/sexplib0 -I /home/opam/.opam/4.11/lib/spawn -I /home/opam/.opam/4.11/lib/splittable_random -I /home/opam/.opam/4.11/lib/stdio -I /home/opam/.opam/4.11/lib/time_now -I /home/opam/.opam/4.11/lib/timezone -I /home/opam/.opam/4.11/lib/typerep -I /home/opam/.opam/4.11/lib/variantslib -no-alias-deps -open Simlog__ -o lib/.simlog.objs/byte/simlog__Thread.cmo -c -impl lib/thread.ml)
# File "lib/thread.ml", line 3, characters 17-41:
# 3 | let get () : t = Caml_threads.Thread.self () |> Caml_threads.Thread.id
#                      ^^^^^^^^^^^^^^^^^^^^^^^^
# Error: Unbound module Caml_threads
# (cd _build/default && /home/opam/.opam/4.11/bin/ocamlc.opt -w -40 -g -bin-annot -I lib/.simlog.objs/byte -I /home/opam/.opam/4.11/lib/base -I /home/opam/.opam/4.11/lib/base/base_internalhash_types -I /home/opam/.opam/4.11/lib/base/caml -I /home/opam/.opam/4.11/lib/base/md5 -I /home/opam/.opam/4.11/lib/base/shadow_stdlib -I /home/opam/.opam/4.11/lib/base_bigstring -I /home/opam/.opam/4.11/lib/base_quickcheck -I /home/opam/.opam/4.11/lib/bin_prot -I /home/opam/.opam/4.11/lib/bin_prot/shape -I /home/opam/.opam/4.11/lib/core -I /home/opam/.opam/4.11/lib/core/error_checking_mutex -I /home/opam/.opam/4.11/lib/core_kernel -I /home/opam/.opam/4.11/lib/core_kernel/base_for_tests -I /home/opam/.opam/4.11/lib/core_kernel/caml_unix -I /home/opam/.opam/4.11/lib/core_kernel/flags -I /home/opam/.opam/4.11/lib/core_kernel/version_util -I /home/opam/.opam/4.11/lib/core_unix -I /home/opam/.opam/4.11/lib/fieldslib -I /home/opam/.opam/4.11/lib/jane-street-headers -I /home/opam/.opam/4.11/lib/ocaml/threads -I /home/opam/.opam/4.11/lib/ocolor -I /home/opam/.opam/4.11/lib/parsexp -I /home/opam/.opam/4.11/lib/ppx_assert/runtime-lib -I /home/opam/.opam/4.11/lib/ppx_bench/runtime-lib -I /home/opam/.opam/4.11/lib/ppx_compare/runtime-lib -I /home/opam/.opam/4.11/lib/ppx_enumerate/runtime-lib -I /home/opam/.opam/4.11/lib/ppx_expect/collector -I /home/opam/.opam/4.11/lib/ppx_expect/common -I /home/opam/.opam/4.11/lib/ppx_expect/config -I /home/opam/.opam/4.11/lib/ppx_expect/config_types -I /home/opam/.opam/4.11/lib/ppx_hash/runtime-lib -I /home/opam/.opam/4.11/lib/ppx_here/runtime-lib -I /home/opam/.opam/4.11/lib/ppx_inline_test/config -I /home/opam/.opam/4.11/lib/ppx_inline_test/runtime-lib -I /home/opam/.opam/4.11/lib/ppx_module_timer/runtime -I /home/opam/.opam/4.11/lib/ppx_sexp_conv/runtime-lib -I /home/opam/.opam/4.11/lib/sexplib -I /home/opam/.opam/4.11/lib/sexplib/unix -I /home/opam/.opam/4.11/lib/sexplib0 -I /home/opam/.opam/4.11/lib/spawn -I /home/opam/.opam/4.11/lib/splittable_random -I /home/opam/.opam/4.11/lib/stdio -I /home/opam/.opam/4.11/lib/time_now -I /home/opam/.opam/4.11/lib/timezone -I /home/opam/.opam/4.11/lib/typerep -I /home/opam/.opam/4.11/lib/variantslib -no-alias-deps -open Simlog__ -o lib/.simlog.objs/byte/simlog__Printer.cmo -c -impl lib/printer.ml)
# File "lib/printer.ml", line 20, characters 16-41:
# 20 |     let mutex = Caml_threads.Mutex.create ()
#                      ^^^^^^^^^^^^^^^^^^^^^^^^^
# Error: Unbound module Caml_threads
# (cd _build/default && /home/opam/.opam/4.11/bin/ocamlc.opt -w -40 -g -bin-annot -I lib/.simlog.objs/byte -I /home/opam/.opam/4.11/lib/base -I /home/opam/.opam/4.11/lib/base/base_internalhash_types -I /home/opam/.opam/4.11/lib/base/caml -I /home/opam/.opam/4.11/lib/base/md5 -I /home/opam/.opam/4.11/lib/base/shadow_stdlib -I /home/opam/.opam/4.11/lib/base_bigstring -I /home/opam/.opam/4.11/lib/base_quickcheck -I /home/opam/.opam/4.11/lib/bin_prot -I /home/opam/.opam/4.11/lib/bin_prot/shape -I /home/opam/.opam/4.11/lib/core -I /home/opam/.opam/4.11/lib/core/error_checking_mutex -I /home/opam/.opam/4.11/lib/core_kernel -I /home/opam/.opam/4.11/lib/core_kernel/base_for_tests -I /home/opam/.opam/4.11/lib/core_kernel/caml_unix -I /home/opam/.opam/4.11/lib/core_kernel/flags -I /home/opam/.opam/4.11/lib/core_kernel/version_util -I /home/opam/.opam/4.11/lib/core_unix -I /home/opam/.opam/4.11/lib/fieldslib -I /home/opam/.opam/4.11/lib/jane-street-headers -I /home/opam/.opam/4.11/lib/ocaml/threads -I /home/opam/.opam/4.11/lib/ocolor -I /home/opam/.opam/4.11/lib/parsexp -I /home/opam/.opam/4.11/lib/ppx_assert/runtime-lib -I /home/opam/.opam/4.11/lib/ppx_bench/runtime-lib -I /home/opam/.opam/4.11/lib/ppx_compare/runtime-lib -I /home/opam/.opam/4.11/lib/ppx_enumerate/runtime-lib -I /home/opam/.opam/4.11/lib/ppx_expect/collector -I /home/opam/.opam/4.11/lib/ppx_expect/common -I /home/opam/.opam/4.11/lib/ppx_expect/config -I /home/opam/.opam/4.11/lib/ppx_expect/config_types -I /home/opam/.opam/4.11/lib/ppx_hash/runtime-lib -I /home/opam/.opam/4.11/lib/ppx_here/runtime-lib -I /home/opam/.opam/4.11/lib/ppx_inline_test/config -I /home/opam/.opam/4.11/lib/ppx_inline_test/runtime-lib -I /home/opam/.opam/4.11/lib/ppx_module_timer/runtime -I /home/opam/.opam/4.11/lib/ppx_sexp_conv/runtime-lib -I /home/opam/.opam/4.11/lib/sexplib -I /home/opam/.opam/4.11/lib/sexplib/unix -I /home/opam/.opam/4.11/lib/sexplib0 -I /home/opam/.opam/4.11/lib/spawn -I /home/opam/.opam/4.11/lib/splittable_random -I /home/opam/.opam/4.11/lib/stdio -I /home/opam/.opam/4.11/lib/time_now -I /home/opam/.opam/4.11/lib/timezone -I /home/opam/.opam/4.11/lib/typerep -I /home/opam/.opam/4.11/lib/variantslib -no-alias-deps -open Simlog__ -o lib/.simlog.objs/byte/simlog__Time.cmo -c -impl lib/time.ml)
# File "lib/time.ml", line 4, characters 16-29:
# 4 | let to_string = to_string_utc
#                     ^^^^^^^^^^^^^
# Error: Unbound value to_string_utc
# Hint: Did you mean to_string_abs?
```